includes/talks/functions.php

```diff
@@ -1191,179 +1191,6 @@ function wct_talks_update_talk() {
 	exit();
 }
 
-/** Sticky talks **************************************************************/
-
-/**
- * Gets the sticky talks
- *
- * @package WordCamp Talks
- * @subpackage talks/functions
- *
- * @since 1.0.0
- *
- * @return array the list of IDs of sticked to front talks
- */
-function wct_talks_get_stickies() {
-	$sticky_talks = get_option( 'sticky_talks', array() );
-
-	/**
-	 * @param  array $sticky_talks the talks sticked to front archive page
-	 */
-	return apply_filters( 'wct_talks_get_stickies', $sticky_talks );
-}
-
-/**
- * Edit WP_Query posts to append sticky talks
- *
- * Simply a "copy paste" of how WordPress deals with sticky posts
- *
- * @package WordCamp Talks
- * @subpackage talks/functions
- *
- * @since 1.0.0
- *
- * @param  array    $posts The array of retrieved posts.
- * @param  WP_Query The WP_Query instance
- * @return array the posts with stickies if some are found
- */
-function wct_talks_stick_talks( $posts = array(), $wp_query = null ) {
```

Comment:
I can't see any instances of `wct_talks_stick_talks` in the codebase now, the only references to `stick` are now in the translation files